README.md

```diff
@@ -1 +1,86 @@
-# webpack-exercise
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Sigma-Quotes
+
+> In this project we've created a dynamic website using two seperate API's to display a list of some quotes along with the ability to like them and leave comments in a dynamically loaded stylish popup.
+
+[Project Video](https://drive.google.com/file/d/1Yj6dQN7wGpV2p7y_UtH5XdKgyHX4TVCI/view?usp=sharing)
```

Comment:
- [ ] Great video presentation. The project requirements require highlighting a piece of code or something you are proud of. I see that you mention a function that checks how many cards are being displayed, kindly try to show the code of this function or the background which changes the color from time to time. Also, try to be more professional. Introducing yourselves is a good way to start a presentation.

<img width="992" alt="image" src="https://user-images.githubusercontent.com/61919617/187952961-5dc7ae64-4616-4d4c-930a-ea91ca7bd868.png">

Reply:
we will redo the video but we will not be creating a new pull request as you stated it is optional and we have already begun the review  process